nautobot/ipam/forms.py

```diff
@@ -703,23 +712,40 @@ class Meta:
             "tags",
         ]
         help_texts = {
-            "location": "Leave blank if this VLAN spans multiple locations",
+            "locations": "Leave blank if this VLAN spans multiple locations",
             "vlan_group": "VLAN group (optional)",
             "vid": "Configured VLAN ID",
             "name": "Configured VLAN name",
             "status": "Operational status of this VLAN",
             "role": "The primary function of this VLAN",
         }
 
+    def clean(self):
+        try:
+            return super().clean()
+        except ValidationError as e:
+            print("\n\n=====> VALIDATION ERROR")
+            raise forms.ValidationError(e.message_dict)
+            
+
 
 class VLANBulkEditForm(
     TagsBulkEditFormMixin,
-    LocatableModelBulkEditFormMixin,
     StatusModelBulkEditFormMixin,
     RoleModelBulkEditFormMixin,
     NautobotBulkEditForm,
 ):
     pk = forms.ModelMultipleChoiceField(queryset=VLAN.objects.all(), widget=forms.MultipleHiddenInput())
+    # TODO (timizuo): Should we have locations in VLAN bulk create
```

Comment:
In bulk edit, it'd be appropriate to have `locations_add` and `locations_remove` fields, similar to how we do in `TagsBulkEditFormMixin`.